src/components/Match/matchColumns.jsx

```diff
@@ -33,8 +33,9 @@ export const heroTd = (row, col, field, index, hideName, party, showPvgnaGuide =
     confirmed={row.account_id && row.name}
     party={party}
     heroName={heroes[row.hero_id] ? heroes[row.hero_id].localized_name : strings.general_no_hero}
-    showPvgnaGuide={showPvgnaGuide}
-    pvgnaGuideInfo={row.pvgnaGuide}
+    showGuide={showPvgnaGuide}
```

Comment:
shouldn't you just pass all of these in as props?

Reply:
i.e., this should probably be implemented to not be pvgna-specific

Reply:
This is on match page. I feel like we can update it if necessary.